### ISSUE TITLE
Publish vertx typescript typings registry

### ISSUE THREAD

**Issue by DanielJoyce** [michel-kraemer/vertx-lang-typescript#6]
That way people can install it via the typings utility under npm

**Comment by michel-kraemer** [comment]
Good idea. I'll have a look at the requirements and will try to understand how to contribute to this project. I'll keep you updated.

**Comment by aristotll** [comment]
Any update for this issue? Thanks.
